firmware/tunerstudio/rusefi.input

```diff
@@ -3823,6 +3835,7 @@ cmd_set_engine_type_default					= "@@TS_IO_TEST_COMMAND_char@@@@ts_command_e_TS_
 		field = "Cranking base IAC position",			crankingIACposition
 		field = "After cranking IAC taper duration",afterCrankingIACtaperDuration
```

Comment:
add a condition here to disable this setting when the new switch is turned on

Reply:
We still use the `afterCrankingIACtaperDuration` or do you mean something else?

Reply:
oop, yeah, it's a multiplier on that duration, not just the duration. lgtm.